src/components/Banner/VoteBanner.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {
+  BannerContainer,
+  Section,
+  Header,
+  Body,
+  ButtonContainer,
+  MainButton,
+  SecondaryButton,
+} from './VoteBanner.style';
+import ExternalLink from 'components/ExternalLink';
+
+const renderSecondaryButton = (redirect: string, cta: string, i: number) => {
+  const Button = i === 0 ? MainButton : SecondaryButton;
+
+  return (
+    <ExternalLink href={redirect}>
+      <Button
+        variant="contained"
+        color="primary"
+        disableRipple
+        disableFocusRipple
+      >
+        {cta}
+      </Button>
+    </ExternalLink>
+  );
```

Comment:
I think you are missing a key here 

![image](https://user-images.githubusercontent.com/114084/93831688-408b8c00-fc28-11ea-9b53-172d4d8b6cc1.png)